dspace-oai/src/main/java/org/dspace/xoai/services/impl/resources/functions/StringXSLFunction.java

```diff
@@ -74,7 +74,6 @@ final public XdmValue call(XdmValue[] xdmValues) throws SaxonApiException {
             log.warn("Empty value in call of function of StringXslFunction type");
             val = "";
         }
-
```

Comment:
Why?

Reply:
It looks better :)

Reply:
I added line but I got checkstyle violation so I had to remove it again....